packages/core/src/hooks/useQueryParamState.ts

```diff
@@ -58,36 +58,37 @@ type SetQueryParams<T> = (params: T) => void;
 
 export function useQueryParamState<T>(
   stateName: string,
-  debounceTime: number = 100,
+  /** @depracated Don't configure a custom debouceTime */
+  debounceTime: number = 250,
```

Comment:
This was also 100 before, now it's 250? Should we restore it back to 100? 😅

Reply:
Not related to the fix, but the UI feels less sluggish when using a higher debounce time. Tradeoff of a higher time is that the URL isn't updated before the user is performing a navigation operation. But 250ms is still quite low.

Reply:
Cool ok :shipit: